internal/pkg/agent/install/install.go

```diff
@@ -166,6 +166,13 @@ func Install(cfgFile, topPath string, unprivileged bool, log *logp.Logger, pt *p
 
 	// install service
 	pt.Describe("Installing service")
+	// ensure that service is removed
+	err = EnsureServiceRemoved(30*time.Second, 250*time.Millisecond, paths.ServiceName())
+	if err != nil {
+		// error context already added by EnsureServiceRemoved
+		pt.Describe(err.Error())
```

Comment:
I think we could use some additional context here, saying something like "failed to ensure service is removed". Otherwise, the progress tracker would say something like:
```
Installing service
failed to connect to service manager
```
IMO, it's not obvious that the second line comes from Agent ensuring that the service is removed.

Reply:
more than a fair point @ycombinator , we should be able to distinguish them apart. I did push this 389cb5aeb19cb24d6e808313f2af87312b9b5ea1 I will wait for the CI to become green and the I will resolve this. That said, please feel free to mention/unresolve if you spot anything else 🙂